sdgym/dataset_explorer.py

```diff
@@ -240,12 +276,19 @@ def _validate_output_filepath(self, output_filepath):
 
         Raises:
             ValueError:
-                If the provided path is not None and does not end with '.csv'.
+                If the provided path is not None and does not end with '.csv', or
+                if the target file already exists.
         """
-        if output_filepath and not Path(output_filepath).suffix == '.csv':
-            raise ValueError(
-                f"The 'output_filepath' has to be a .csv file, provided: '{output_filepath}'."
-            )
+        if output_filepath:
+            path_obj = Path(output_filepath)
+            if path_obj.suffix != '.csv':
+                raise ValueError(
+                    f"The 'output_filepath' has to be a .csv file, provided: '{output_filepath}'."
+                )
+            if path_obj.exists():
+                raise ValueError(
+                    f"The file '{path_obj}' already exists. Please provide a new 'output_filepath'."
+                )
```

Comment:
can we move this to a `validate_output_path` function since it is doing now multiple checks